src/MLIAP/mliap_so3.cpp

```diff
@@ -374,23 +374,20 @@ void MLIAP_SO3::compute_W(int nmax, double *arr)
   double *sqrtD = new double[n * n];
   double *tempM = new double[n * n];
 
-  double **temparr;
-  double tempout[n];
```

Comment:
@macstein FYI, this and the two cases of variable length arrays below slipped through our automated tests. These "variable length" arrays are not allowed by C++ (only in C98). In C++ "n" would have to be a compile time constant.